### DISCUSSION TITLE
Add ability to scatter-plot impedances on Smith chart

### DISCUSSION THREAD

**Discussion by BlankAdventure** [scikit-rf/scikit-rf#1060] [Ideas]
All examples I've seen of Smith chart usage has been to plot frequency-dependent reflection coefficients from network objects, with connected lines, e.g.:

![image](https://github.com/scikit-rf/scikit-rf/assets/24900496/b9afdc1e-4b44-4e61-b38c-66d122a28dcc)

However I've often wanted to make Smith chart plots such as this:

![image](https://github.com/scikit-rf/scikit-rf/assets/24900496/60cf837f-b4fe-446b-b464-604fae2f21e1)

where the dots represent individual impedances before and after a matching circuit. As far as I can tell, there is no straightforward way to do this currently (the above figure was created in plotly. Still not happy that I was stuck with normalized values)

I'd be open to working on this!

**Comment by mhuser** [comment]
Hello @BlankAdventure ,
[In this example](https://scikit-rf.readthedocs.io/en/latest/tutorials/Plotting.html#Advanced-Smith-Chart-with-markers) you can see how to draw a blank smith chart and how to use scatter to plot markers on it.
Maybe your impedances could be plotted in this way?

**Reply by BlankAdventure** [reply]
Ah, thanks, that helped. Here's a minimal example if anyone else is interested:

```python

import skrf as rf
import matplotlib.pyplot as plt
import numpy as np


# Convert impedance to reflection coefficient
def refco(zlist: list, z0: float=50) -> list[complex]:
    return [(z - z0) / (z + z0) for z in zlist]

# prepare figure
fig, ax = plt.subplots(1, 1, figsize=(7,8))

# impedance smith chart
rf.plotting.smith(ax = ax, draw_labels = True, ref_imm = 50.0, chart_type = 'z')

z = [10+10j, 25-25j]

rc = refco(z)

#Plot unconnected dots 
ax.scatter(x=np.real(rc), y=np.imag(rc), marker = 'v', s=20, color='red')
```

**Reply by mhuser** [reply]
Very nice!
![image](https://github.com/scikit-rf/scikit-rf/assets/8080934/3c57e05b-473d-48be-8cbb-9193929708ff)
